### PR TITLE
publish site for 2017.18

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Publish site for 2017.18

## Type of Pull Request:

- [ ] Blog Post
- [x] Content (Documentation updates or typo-fixes)
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-125165](https://oktainc.atlassian.net/browse/OKTA-125165)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested the change across Chrome, Firefox, Safari, and mobile.

## Primary Reviewers:

- *Tag reviewer(s)*
@jmelberg-okta 

## Before Merging:

- [ ] I have had this PR reviewed and approved on GitHub.
- [x] I have squashed my commits into `1` commit or will use the **Squash and Merge button**.

### CODE REVIEW

File: docs/api/resources/factor_admin.html [hunk 1]
```diff
@@ -111,7 +111,7 @@ <h3 class="Sidebar-title">Use Cases</h3>
               <a href="/use_cases/api_access_management">API Access Management</a>
             </li>
             <li>
-              <a href="/use_cases/promote-oan-integration">Promote Your Integration</a>
+              <a href="/use_cases/integrate_with_okta">Integrate with Okta</a>
```
Comment (jmelberg-okta) [inline docs/api/resources/factor_admin.html]: Can you confirm that`http://localhost:4000/docs/api/getting_started/factor_admin.html` returns the page?

I'm returning a `404`

File: docs/api/resources/system_log.html [hunk 2]
```diff
@@ -1234,40 +1234,227 @@ <h3 id="ipaddress-object">IpAddress Object</h3>
...
+  <li><code class="highlighter-rouge">policy.evaluate_sign_on</code> provides context on the values used and evaluated in the context of the Okta sign on policy. For example, you can determine which network zones were matched for this event.</li>
+  <li>For <code class="highlighter-rouge">policy.lifecycle</code> and <code class="highlighter-rouge">policy.rule</code> events, the corresponding policy is listed in the target object.</li>
+</ul>
+
+<h3 id="user-events">User Events</h3>
+
+<table>
+  <thead>
+    <tr>
+      <th style="text-align: left">Event</th>
```
Comment (jmelberg-okta) [inline docs/api/resources/system_log.html]: nit: The table column formatting is different here than the rest of the page.

File: use_cases/integrate_with_okta/promotion.html [hunk 3]
```diff
@@ -0,0 +1,392 @@
+<!DOCTYPE html>
```
Comment (jmelberg-okta) [inline use_cases/integrate_with_okta/promotion.html]: Is this the intended format?

Comment (mystiberry-okta) [inline use_cases/integrate_with_okta/promotion.html]: Yes. Using headers made the structure hard to understand.

File: docs/api/resources/system_log.html (hunk 2 above)
Comment (mystiberry-okta) [inline docs/api/resources/system_log.html]: Great catch. I'll fix that in the next iteration (we're adding more to this section soon).

File: docs/api/resources/factor_admin.html (hunk 1 above)
Comment (mystiberry-okta) [inline docs/api/resources/factor_admin.html]: we saw that was fixed.

Comment (mystiberry-okta) [inline docs/api/resources/factor_admin.html]: That page should be /docs/api/resources/factor_admin.html, which currently returns 404 in published site. I'll dig into this tomorrow, it's an existing issue, not sure how this beta doc disappeared, but it doesn't belong in /getting_started.

**Pull request closed without merging**